debug.log

```diff
@@ -0,0 +1,2 @@
+[1113/103803.875:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Included the error log intentionally or not?

Reply:
ohh this is a mistake, good catch